index.d.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference types="jest"/>
```

Comment:
It's still in [docs](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html)